README.md

```diff
@@ -1 +1,25 @@
 # es-imputation-sg
```

Comment:
Doesn't include means method but this is in the means feature branch.

Reply:
Fair point, but you wrote the means method and wrangler, we only wrote the tests :-)

Reply:
Yeah it's cool, just wanted to add a note so that I know it was to be merged in.